static/styles/zulip.css

```diff
@@ -1465,6 +1455,15 @@ td.pointer {
     opacity: 1;
 }
 
+.message_header + .selected_message {
+    /* Sticky message header overlaps with the box-shadow, so we add another box-shadow to
+       compensate for that. */
```

Comment:
Can you tweak this to explain a bit more how it compensates?

Reply:
Done.